### PR TITLE
Make dockerUtils use container name for test exec

### PR DESCRIPTION
**Pull request by aahmed-se**
Provides easier to follow logs also we reduce the log output via custom log config for the integration tests.

### CODE REVIEW

File: tests/integration/src/test/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sijie) [inline tests/integration/src/test/resources/log4j2.xml]: why do we need this?

the logging is already configured - https://github.com/apache/incubator-pulsar/blob/master/buildtools/src/main/resources/log4j2.xml

you can enable logging to console by specifying `-DredirectTestOutputToFile=false`

https://github.com/apache/incubator-pulsar/blob/master/pom.xml#L132

**Comment by aahmed-se** [comment]
We are overriding  that , we are reducing the number of log line elements since we are not looking at multithreaded server components but single threaded test instances, also we are making the log files easier to follow bu using the containername

**Comment by sijie** [comment]
>  we are reducing the number of log line elements since we are not looking at multithreaded server components but single threaded test instances

well, I think it is debatable. more information is generally helping with debugging things. is there any particular reasons that you have to reduces the log line elements?

> also we are making the log files easier to follow bu using the containername

how does the log configure help that?

**Comment by sijie** [comment]
> the log configure does not, the DockerUtils changes fixes that issue.

can you please comment inline and address the comment individually? because my comments are for different places. especially for the log configuration file, I don't see it is really needed.

**Pull request closed without merging**